AztecTests/TextViewTests.swift

```diff
@@ -1516,4 +1516,24 @@ class TextViewTests: XCTestCase {
 
         XCTAssertEqual(html, "")
     }
+
+    func testParseImageWithExtraAttributes() {
```

Comment:
To keep in line with the other tests, you should document the test cases explaining at least in high-level what is going on inside the test.